data/mods/gen2/moves.ts

```diff
@@ -415,7 +415,7 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 	},
 	mimic: {
 		inherit: true,
-		accuracy: true,
+		accuracy: 100,
```

Comment:
You can just remove this altogether; it'll inherit the 100 from Gen 3.

Reply:
Yep, thanks! I have removed it